### PR TITLE
known amendments - ticksize and cryptoconditions

### PR DESCRIPTION
**Pull request by mDuo13**
Add TickSize and CryptoConditions to the list of known amendments. (The description of CryptoConditions is kind of a placeholder for now.)

Edited the Tickets and CryptoConditions "introduced version" to say "N/A" since (to my knowledge) neither amendment is considered final/stable right now.

### CODE REVIEW

**Comment by mDuo13** [comment]
per @JoelKatz elsewhere, this is accurate

File: content/concept-amendments.md [hunk 1]
```diff
@@ -238,6 +251,17 @@ Allows pre-authorization of accounting relationships (zero-balance trust lines)
...
 
+## TickSize
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 532651B4FD58DF8922A49BA101AB3E996E5BFBF95A913B3E392504863E63B164 | Released but not enabled |
+
+Changes the way Offers are ranked in order books, so that currency issuers can configure how many significant digits are taken into account when ranking Offers by exchange rate. With this amendment, the exchange rates of Offers are rounded to the configured number of significant digits, so that more Offers have the same exact exchange rate. The intent of this change is to require a meaningful improvement in price to outrank a previous Offer. If used by major issuers, this should reduce the incentive to spam the ledger with Offers that are only a tiny fraction of a percentage point better than existing offers. It may also increase the efficiency of order book storage in the ledger, because Offers can be grouped into fewer exchange rates.
```
Comment (ryangyoung) [inline content/concept-amendments.md]: > Offers

Maybe add a link to https://ripple.com/build/transactions/#lifecycle-of-an-offer

File: content/concept-amendments.md [hunk 2]
```diff
@@ -238,6 +251,17 @@ Allows pre-authorization of accounting relationships (zero-balance trust lines)
...
+
+Introduces a `TickSize` field to accounts, which can be set with the AccountSet transaction type. If a currency issuer sets the `TickSize` field, the Ripple Consensus Ledger truncates the exchange rate (ratio of funds in to funds out) of Offers to trade the issuer's currency, and adjusts the amounts of the Offer to match the truncated exchange rate. If only one currency in the trade has a `TickSize` set, that number of significant digits applies. When trading two currencies that have different `TickSize` values, whichever `TickSize` indicates the fewest significant digits applies. XRP does not have a `TickSize`.
```
Comment (ryangyoung) [inline content/concept-amendments.md]: > AccountSet transaction type

Probably should add a link to: https://ripple.com/build/transactions/#accountset

**Review by ryangyoung — APPROVED** [review]
Saw a couple of places where links to the reference docs would be helpful, but otherwise looks great to me. +2

**Pull request merged**